src/oc-id/config/settings.yml

```diff
@@ -1,5 +1,5 @@
 chef:
-  endpoint: https://192.168.33.100
+  endpoint: https://33.33.33.10
```

Comment:
Yeah, my bad.  I didn't mean to check that in.  I have to change it every time because I never managed to get dvm working.